src/components/nodes/logical/And.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div/>
+</template>
+
+<script>
+export default {
+  spec: {
+    id: 'std::and',
+    title: 'And',
+    icon: require('../../../assets/node_icons/node_conjunction.svg'),
+    inputs: [
+      {
+        id: 'boolean',
+        name: 'Boolean',
+        type: 'boolean',
+        variadic: {
+          minimum: 2
+        }
+      }
+    ],
+    outputs: [
+      {
+        id: 'result',
+        name: 'Result',
+        type: 'boolean'
+      }
+    ],
+    calculate (input) {
+      const booleans = input.boolean.filter(booleans => !isNaN(booleans))
```

Comment:
Checking for `NaN` on bools seems a little counterintuitive to me. The arithmetic nodes basically only use this to rule out `undefined` (on variadics) so you could alternatively just directly check for `undefined` or use some other method to detect all bools (and especially anything castable to bools).
Thus, there are valid inputs to this node which would be ignored using this method, namely any string not resembling a number (e.g. 'kek'), these are `NaN` but should be castable to a bool (and are allowed to be in our current casting rules).

This comment applies to all the other nodes using this filter as well.